mbed_targets/_internal/target_database.py

```diff
@@ -72,3 +90,14 @@ def _response_error_code_to_str(response: requests.Response) -> str:
         )
     else:
         return f"An HTTP {response.status_code} was received from '{_TARGET_API}' containing:\n{response.text}"
+
+
+def _get_request() -> requests.Response:
+    """Make a get request to the API, ensuring the correct headers are set."""
+    token = os.getenv(_AUTH_TOKEN_ENV_VAR)
+    if not token:
```

Comment:
I think it is simpler to to invert the logic of this if statement and remove the `not`.

Reply:
Agree; flip that logic!